scripts/events_module/relationship/pregnancy_events.py

```diff
@@ -714,7 +714,7 @@ def get_second_parent(cat, clan):
                         if not p_rel.opposite_relationship:
                             p_rel.link_relationship()
                         p_rel_opp = p_rel.opposite_relationship
-                        if p_rel_opp.like < -20 and p_rel.like < -20:
+                        if not p_rel_opp.like < -20 and not p_rel.like < -20:
```

Comment:
I think this would be easier to read if we just flipped the `<`, so:

```suggestion
                        if p_rel_opp.like > -20 and p_rel.like >  -20:
```